src/Visualizer/.eslintrc.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
same question as https://github.com/Samsung/ONE-vscode/pull/93#discussion_r709817145

Reply:
When eslint encounters `package.json` or `.esconfig.json` during the `npm run lint`, the nearest file is overriding. There is a problem with the eslintconfig set in the `Visualizer/package.json` , so we will remove the setting and remove the `Visualizer/.eslintrc.json`

Thank you for the feedback! 🙏